Lib/test/test_threading.py

```diff
@@ -1338,6 +1339,20 @@ def run(self):
         # explicitly break the reference cycle to not leak a dangling thread
         thread.exc = None
 
+    def test_multithread_modify_file_noerror(self):
```

Comment:
```suggestion
    def test_multithread_modify_file_noerror(self):
        # See issue25872
```

Reply:
I add a comment like this when the test is a little strange, because it helps explain where the test came from. For standard unit tests like the other ones it's not necessary.